gcn/notices/core/Localization.schema.json

```diff
@@ -19,14 +19,18 @@
       "default": "circle"
     },
     "ra_uncertainty": {
-      "type": "array",
-      "maxItems": 2,
-      "description": "Uncertainty in RA [deg] with optional asymmetric errors, measured from the RA in the schema; respective ex: [r], [r1], [l1-, l1+]. For ellipse, ra axis is r1 or semi-major axis."
+      "description": "Uncertainty in RA [deg], measured w.r.t. RA. A single number for circle, and ellipse, RA axis alligns with semi-major axis. For rectangle, use optional aymmetric errors array, [l1-, l2+].",
```

Comment:
What does "Uncertainty in RA measured with respect to RA" mean?

Reply:
So there have been confusion, about it's understanding. It's uncertainty error measured from RA centre like 2 (-+0.2) or, shape coordinates 2 (1.8, 2.2), may be you can help with better wordings.

Reply:
In that case, the name of this field is wrong. It is the uncertainty in the semimajor axis.

Reply:
But but if it's rectangle?